gc/base/TLHAllocationSupport.cpp

```diff
@@ -48,6 +48,7 @@
 #include "MemorySubSpace.hpp"
 #include "ObjectAllocationInterface.hpp"
 #include "ObjectHeapIteratorAddressOrderedList.hpp"
+#include "ParallelGlobalGC.hpp"
```

Comment:
Do we need this include? The concern is TLH code usage is not limited to `MM_ParallelGlobalGC` (for example can be used in Balanced). I guess the problem is solved already by moving helper to `MM_GlobalCollector`, is not it?

Reply:
Yup the include is stale and was leftover after moving the helper out of `MM_ParallelGlobalGC`